Yafc.Model/Model/Project.cs

```diff
@@ -148,6 +148,29 @@ public class Project : ModelObject {
             page.MarkAsDeleted();
             _ = this.RecordUndo().pages.Remove(page);
         }
+
+        /// <summary>
+        /// Get the page that is visually next (i.e. to the right of the current selected page on the tab bar)
+        /// from the specified one.
+        /// </summary>
+        /// <param name="currentPage">The page to get the next page from (probably the current page).</param>
+        /// <returns>The page object that should be set to active.</returns>
+        public ProjectPage? VisibleNeighborOfPage(ProjectPage? currentPage, bool forward = true) {
```

Comment:
Does it make sense to let `forward` have a default? I am not saying it is bad, but wonder the benefit :thinking:

Reply:
Fair enough, I think it would be fine without one. I also need to document it, oops.

Reply:
Is it more common to use a bool, an enum, or an int that's implicitly +-1 in codebases like this for this sort of thing?

Reply:
bool and enum are both used in different places IIRC.
int feels a bit finicky as it could be +-2...

Reply:
I think a bool is good for the situation where there is an On/Off switch, and it's clear to a new reader what this switch is used for.
If it's unclear, then I'd make an enum with explanation of its purpose and the docs for its members.

In this case, I think bool is good.